smf/zone-network-setup/manifest.xml

```diff
@@ -6,21 +6,26 @@
 <service name='oxide/zone-network-setup' type='service' version='1'>
   <create_default_instance enabled='true' />
 
-  <!-- Run after the operating system's svc:/network/physical service is done. -->
+  <!-- Run after the zone's networking stack is up. -->
   <dependency name='physical' grouping='require_all' restart_on='none'
     type='service'>
-  <service_fmri value='svc:/network/physical:default' />
+  <service_fmri value='svc:/milestone/network:default' />
   </dependency>
 
-  <dependency name='multi_user' grouping='require_all' restart_on='none'
-    type='service'>
-  <service_fmri value='svc:/milestone/multi-user:default' />
+  <!-- The zone-setup binary is not ready to run until its initial properties
+          have been set by the sled-agent, which happens after the
+          `manifest-import` service is running.
+  -->
+  <dependency name='zone-network-setup-manifest-import' type='service' grouping='require_all' restart_on='none'>
```

Comment:
```suggestion
  <dependency name='manifest-import' type='service' grouping='require_all' restart_on='none'>
```
nit, the name can just be `manifest-import` if you like, and that's the usual convention.
`<dependency/>` clauses are local to this service so the name only needs to be unique within it.
`<dependent/>` clauses, in contrast, are injected into the other (referenced) service as a dependency, and so need a more unique name.

Reply:
Done in 6c93f59a4